CHANGES.md

```diff
@@ -21,6 +21,15 @@ To be released.
     `IImmutableDictionary<string, IValue>`.  [[#3321]]
  -  Removed `EnumerableMeasurement` class.  [[#3325]]
  -  Removed `KeyValueExtensions` class.  [[#3325]]
+ -  Removed `StateStoreExtensions.EncodeKey()` and
+    `StateStoreExtensions.DecodeKey()` methods.  [[#3328]]
+ -  Removed `StateStoreExtensions.GetStates(IStateStore, HashDigest<SHA256>?,
+    IReadOnlyList<string>)` method.  [[#3328]]
+ -  Removed `TrieExtensions.Set(ITrie,
+    IEnumerable<KeyValuePair<string, IValue?>)` method.  [[#3328]]
+ -  Added `KeyBytes.Encoding` static property.  [[#3328]]
+ -  Removed `KeyBytes(string, Encoding)` constructor.  [[#3328]]
+ -  Added `KeyBytes(string)` constructor.  [[#3328]]
```

Comment:
This line should move to the `Added APIs` section. 🤔 Or, since it is in the same context as the sentence above, I want to change it into one sentence and change it to something like `Replaced ...`